release/release.go

```diff
@@ -55,8 +55,24 @@ var (
 // ForceDevMode returns true if the distribution doesn't implement required
 // security features for confinement and devmode is forced.
 func (o *OS) ForceDevMode() bool {
-	level, _ := apparmor.ProbeKernel().Evaluate()
-	return level != apparmor.Full
+	return AppArmorSupportLevel() != apparmor.FullSupport
+}
+
+// AppArmorSupportLevel quantifies how well apparmor is supported on the current kernel.
+func AppArmorSupportLevel() apparmor.SupportLevel {
+	level, _ := aa.SupportLevel()
```

Comment:
Sorry, but it feels super strange to have such trivial helpers to our own trivial helpers (!) that are sitting across two different packages, and having one of those two packages consistently entirely of the trivial helpers.

Since this is taking a bit longer than it should for such a simple change,  here is a more clear path for LGTM:

- The level function becomes release.AppArmorLevel
- The type becomes release.AppArmorLevelType
- The summary becomes release.AppArmorSummary
- All of that inside release/apparmor{,_test}.go
- KernelSupport dies. All of that functionality fits inside a tiny init function that runs once during execution and stores single two global variables: one level, one summary.
- apparmor package dies. Its _whole_ point is that funcionality today.

Reply:
Ah, and variables might be _release.{No,Partial,Full}AppArmor._